onnx/defs/math/defs.cc

```diff
@@ -2586,7 +2586,9 @@ ONNX_OPERATOR_SET_SCHEMA(
              "tensor(int32)",
              "tensor(int64)",
              "tensor(float)",
-             "tensor(double)"},
+             "tensor(double)",
+             "tensor(float16)",
+             "tensor(bfloat16)"},
```

Comment:
This looks good. Will some predefined-list like ```numeric_types_for_math_reduction_with_bfloat()``` work? That will help make sure we don't miss this op if a new type is added in the future.

Reply:
Also, updating description string below will be good (since not all tensor types are allowed).

Reply:
Good suggestion. I've updated it.